config/locales/server.en.yml

```diff
@@ -4,7 +4,7 @@ en:
   site_settings:
     babble_enabled: "Enable Babble Shoutbox plugin."
     babble_placeholder: "Leave blank for default placeholder text in empty chat textarea. Or enter your custom text here."
-    babble_icon: "Font Awesome code (fa- omitted) for the Babble icon in the upper right corner. See http://fontawesome.io/cheatsheet/ for a list of icons."
+    babble_icon: "Font Awesome code (fa- omitted) for the Babble icon in the upper right corner. See http://fontawesome.io/cheatsheet/ for a list of icons. You might have to add the icon to the \"svg icon subset\" site setting as well."
```

Comment:
Hmmm we shouldn't have to tell the user this. Is there a way to dynamically adjust which icons are svg? Is there a fallback / what happens if we don't register these as svg icons?

Also, I don't see a svg_icon_subset setting on the latest discourse; could you point me at the PR where it will be added?

Reply:
There currently isn't a way to do this for a plugin setting, though there is a way for the plugin to register an icon. Hmm, I will see if we can cover that in core, it should be doable. 

Also, more details about the icon change here: https://meta.discourse.org/t/introducing-font-awesome-5-and-svg-icons/101643 

And the branch with these changes is here: https://github.com/discourse/discourse/tree/fontawesome5